onnxruntime/core/optimizer/conv_activation_fusion.cc

```diff
@@ -173,10 +175,15 @@ namespace actions {
 using NTO = NodesToOptimize;
 
 class FuseConvActivationAction : public ReplaceWithNew {
+ public:
+  FuseConvActivationAction(const std::string& domain = kMSDomain) : ReplaceWithNew(), domain_{domain} {}
+
  private:
-  std::string OpType(const RuntimeState&) const override { return "FusedConv"; }
+  std::string OpType(const RuntimeState&) const override {
+    return domain_ == kMSDomain ? "FusedConv" : "Conv";
+  }
 
-  std::string Domain(const RuntimeState&) const override { return kMSDomain; }
+  std::string Domain(const RuntimeState&) const override { return domain_; }
```

Comment:
Can the OpType and Domain can be inferred from the target node so a domain_ member isn't required? Seem like we return kMsDomain/FusedConv unless the target node's domain is the internal NHWC domain, in which case we return kMSInternalNHWCDomain/Conv

Reply:
Modified code to used domain from the target node.

Reply:
The target domain is not set when the Domain() and Optype() methods are called on the target node.